serverless/serverless
# Allow individual packaging with TypeScript source maps

## What did you implement:

Closes #5742 



## How did you implement it:

Add the source maps directly to the handlers so we can use `slsw.lib.entries` in `webpack.config.js` that allows for individual packaging. 



## How can we verify it:

The unit tests pass, however, the integration template tests fail during deploy with `Serverless plugin "serverless-webpack" not found. Make sure it's installed and listed in the "plugins" section of your serverless config file.`. This same failure exists on the master branch, though.



## Todos:


***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO
